mainapp/views.py

```diff
@@ -205,10 +212,49 @@ class RegisterContributor(CreateView):
 class HomePageView(TemplateView):
     template_name = "home.html"
 
+    def get_context_data(self, **kwargs):
+
+        home_page_setting = getattr(settings, 'HOME_PAGE_ANALYTICS', None)
```

Comment:
Do we need to override the global variable here?

Reply:
actually no... i will clear that